src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -8,7 +9,11 @@ import { selectAssets } from 'state/slices/selectors'
 import { TradeState } from '../../Trade'
 import { TradeActions, useSwapper } from '../useSwapper/useSwapper'
 
-export const useTradeRoutes = (): {
+const ETHEREUM_CAIP19 = 'eip155:1/slip44:60'
```

Comment:
Do we have a way to get the ETH caip19 without adding it statically there? I couldn't find except inside mocks constants which is not reliable

Reply:
We're good here, that's the way to go! We have a `toCAIP19` that allows you to get a CAIP19, but all it does is reconstructing a CAIP19 from its parts, so using this just make it more verbose than is necessary.